### DISCUSSION TITLE
Mastodon?

### DISCUSSION THREAD

**Discussion by samwilson** [lib-static/lib-static.github.io#15] [Q&A]
Is there a Mastodon account for lib-static?

**Comment by dcnb** [comment]
There is not, @samwilson. Right now, Lib-static consists of the website and its resources. We don't really have a wider presence than that. (Nor have we seen a huge demand for one ...) 

Do you have an interest in starting one? Or contributing? No worries if not; we're just always happy to have folks be a part of the community. 

cheers, @dcnb

**Comment by evanwill** [comment]
adding to what @dcnb said-- @samwilson we do have a #lib-static channel on code4lib Slack, you can find the sign up at [code4lib chat](https://code4lib.org/irc/). Many of the Lib-Static folks are also active on the #minicomp-wax channel as well, and many of us are on Mastodon as individuals. We (@dcnb and I) are more active as CollectionBuilder maintainers, so feel free to [reach out to us there too](https://collectionbuilder.github.io/cb-docs/#how-to-get-in-touch)!

**Comment by samwilson** [comment]
Thanks! Sounds good.

I guess I was mainly just wondering how much of an 'organisation' there is; not that there needs to be!

I'll check out CollectionBuilder, it sounds terrific. :-)
